apiClient/models.go

```diff
@@ -0,0 +1,34 @@
+package apiClient
```

Comment:
please name the file types.go instead of models. we don't have any models in this project.